pages/emailSender.tsx

```diff
@@ -165,95 +188,110 @@ const EmailSender: NextPage = () => {
 
   return (
     <Layout>
-    <ThemeProvider theme={theme}>
-      <StyledPageContainer>
-        <Typography variant="h3"> Email Sender </Typography>
-        <Divider />
-        <br />
-        <FormControl fullWidth>
-          <SectionContainer>
-            <StyledSubHeader variant="h5">1) Enter message</StyledSubHeader>
-            <br />
-            <StyledTextArea
-              aria-label="message-text-area"
-              placeholder="Paste in message"
-              onChange={(e) => setMessage(e.target.value)}
-              minRows={20}
-            />
-          </SectionContainer>
+      <ThemeProvider theme={theme}>
+        <StyledPageContainer>
+          <Typography variant="h3"> Email Sender </Typography>
+          <Divider />
+          <br />
+          <FormControl fullWidth>
+            <SectionContainer>
+              <StyledSubHeader variant="h5">1) Enter message</StyledSubHeader>
+              <br />
+              <StyledTextArea
+                aria-label="message-text-area"
+                placeholder="Paste in message"
+                onChange={(e) => setMessage(e.target.value)}
+                minRows={20}
+              />
+            </SectionContainer>
+            <SectionContainer>
+              <StyledSubHeader variant="h5">
+                2) Upload and import csv
+              </StyledSubHeader>
+              <StyledCsvButtonsContainer>
+                <input
+                  style={{ display: 'none' }}
+                  id="contained-button-file"
+                  accept={'.csv'}
+                  type="file"
+                  onChange={handleOnChange}
+                />
+                <label htmlFor="contained-button-file">
+                  <Button variant="contained" component="span">
+                    Upload
+                  </Button>
+                </label>
+                <StyledCsvButton
```

Comment:
We should disable this button visually (gray it our or something) if no csv has been uploaded yet